provider/cmd/app/app.go

```diff
@@ -305,6 +308,7 @@ func Run(ctx context.Context, opts Options) error {
 			NetworkInterfacePlugin:   nicPlugin,
 			ResyncIntervalVolumeSize: opts.ResyncIntervalVolumeSize,
 			EnableHugepages:          opts.EnableHugepages,
+			MachineStateSyncInterval: opts.MachineStateSyncInterval,
```

Comment:
please keep it consistent with `	ResyncIntervalVolumeSize: opts.ResyncIntervalVolumeSize,`

Reply:
we created inconsistency here with gc.

Reply:
graceful shutdown isn't in main therefore I would propose to address it in the graceful shutdown PR

Reply:
okay. But i see sourav didn't implement my recommendation. You have different format for variable and arguments. arguments follow pattern general -> specific. Here you follow pattern specific -> general. So you have reverted logic. It looks for me as complication as help.

This is different:
o.ResyncIntervalMachineState, "machine-state-resync-duration"

General -> specific has benefit because you can do grouping of variable base on name which serve for the same logic and you know they have some relationship. It is better extendable in future too.

If you don't want it, you don't have to do implement it.

Reply:
Following our offline discussion, I have updated the key to `ResyncIntervalMachineState`. We can handle the consistent naming convention separately.